src/hooks/useCreateAntModal/useCreateAntModal.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+
+import { useGlobalState } from '../../state/contexts/GlobalState';
+
+export default function useCreateAntModal() {
+  // eslint-disable-next-line
```

Comment:
nit: is this needed?

Reply:
nope, removed

Reply:
can you share the commit? i still see it